tox.ini

```diff
@@ -63,14 +63,13 @@ commands =
     echo "Troubleshooting guide: https://docs.plasmapy.org/en/latest/contributing/doc_guide.html#troubleshooting"
 deps = -r{toxinidir}/requirements.txt
 
-[testenv:build_docs_linkcheck]
+[testenv:linkcheck]
```

Comment:
I settled on this name since I kept typing `tox -e linkcheck` even after getting multiple reminders that this wasn't the name of an environment.  🙃